front/src/applications/operationalStudies/views/ProjectsList/ProjectsListView.tsx

```diff
@@ -31,7 +31,7 @@ type SortOptions =
   | 'LastModifiedAsc'
   | 'LastModifiedDesc';
 
-const HomeOperationalStudies = () => {
+const ProjectsListView = () => {
```

Comment:
Usually in English the singular form of the noun is used when followed by "list".